Bottom: ae021e49d42717db8ff626a6c73e32165a632afb
Top:    ae6374f4f52b047b3f17707bae498916bba862d7
Author: Alexey Lyashkov <shadow@Alexeys-MacBook-Pro.local>
Date:   2016-02-02 14:48:37 +0300

Refresh of ctl-verbs.patch

---

diff --git a/srv.c b/srv.c
index 105d18a..d932969 100644
--- a/srv.c
+++ b/srv.c
@@ -46,7 +46,7 @@ srv_init(void)
 		}
 		if (read_st && (event & POLLIN)) {
 			/* read done */
-			printf("%s\n", data);
+			printk("%s\n", data);
 			kfree(data);
 			read_st = false;
 		}
